Android/skeleton-extension-android/skeletonextension/src/main/java/com/sample/company/extension/SkeletonExtension.java

```diff
@@ -174,6 +197,9 @@ private void processGetterRequestEvent(final Event requestEvent) {
 
         if (responseEvent == null) {
             MobileCore.log(LoggingMode.WARNING, getName(), "An error occurred constructing the response event.");
+            /* Even though the response event is nil, continue to call ACPCore::dispatchResponseEvent as a response still needs
```

Comment:
If we return here a response event will not get dispatched.  If `responseEvent` is null, internally an empty `Event` is created and passed back to the paired callback.